.circleci/config.yml

```diff
@@ -180,17 +181,17 @@ jobs:
 workflows:
   version: 2
   # regular runs of build-docs
-  # bioconda-utils-build-docs:
-  #    triggers:
-  #      - schedule:
-  #          cron: "0 0,6 * * *"
-  #          filters:
-  #            branches:
-  #              only:
-  #                - master
-  #    jobs:
-  #      - build-docs:
-  #          context: org-global
+  bioconda-utils-build-docs:
```

Comment:
Is this change intended to be in this commit?

Reply:
not intended! decided to simplify and refactor though in [1631bf3](https://github.com/bioconda/bioconda-utils/pull/803/commits/1631bf3ea34dc71599f7f95d2dfff21984414f82)